terraform/services/tfstate/variables.tf

```diff
@@ -12,7 +12,7 @@ variable "env" {
   type        = string
   validation {
     condition     = contains(["dev", "test", "sbx", "prod", "mgmt"], var.env)
-    error_message = "Valid value for env is dev, test, sbx, prod, or mgmt."
+    error_message = "Valid value for env is dev, test, sbx, mgmt or prod."
```

Comment:
You'll need to add "sandbox" here to allow for "sbx" in legacy and "sandbox" in greenfield.